ooniprobe/View/RunTestViewController.mm

```diff
@@ -0,0 +1,121 @@
+//
+//  RunTestTableViewController.m
+//  ooniprobe
+//
+//  Created by Lorenzo Primiterra on 06/08/17.
+//  Copyright © 2017 Simone Basso. All rights reserved.
+//
+
+#import "RunTestViewController.h"
+
+@interface RunTestViewController ()
```

Comment:
@lorenzoPrimi I have seen this pattern frequently used, but I don't fully understand it. Why one needs to repeat the interface again in the implementation file just leaving it empty?

Reply:
It's the default pattern when you create a new class, I never asked myself why